### PR TITLE
Replace div with button for handle

### LINKED ISSUE #25: Slider is not accessible via keyboard
Pressing `tab` on `Shared room checkbox` focuses `More filters`. Similarly pressing `shift + tab` on `More filters` focuses `Shared room checkbox`. Skipping the slider control in both forward and backward navigation.

Please find the stated behavior in the below animation.

slider animation

For expected keyboard interaction, please refer [WAI-ARIA Authoring Practices - Slider (Multi-Thumb)](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/#slidertwothumb).

And I guess for the `slider button` you would prefer to use`<button>` instead of `<div>`. Please check out [Use button if you mean it](https://sarbbottam.github.io/accessibility,/javascript/2016/06/11/use-button-if-you-mean-it/) for further details.

screenshot of slider markup

### PR DESCRIPTION
**Pull request by goatslacker**
Fixes #25 

@ljharb

### CODE REVIEW

File: src/Slider.jsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ export default React.createClass({
       algorithm: linear,
       className: '',
       disabled: false,
-      handle: 'div',
+      handle: 'button',
```
Comment (ljharb) [inline src/Slider.jsx]: since the default "type" of `<button>` is "submit", we need to ensure this uses an explicit type of "button".

Comment (goatslacker) [inline src/Slider.jsx]: Done. Now it uses a component as a default

**Comment by ljharb** [comment]
LGTM. I can't decide if this is a breaking change or semver-minor - but I think breaking?

**Comment by goatslacker** [comment]
Hmmm, shouldn't impact anything user related (negatively) that I know of.

**Comment by ljharb** [comment]
Unless someone is doing something bad like targeting it with CSS - or unless they've written tests that target a div.

**Comment by goatslacker** [comment]
That is true, however, is that really part of the "public API"? Is it reasonable to expect that level of support?

**Comment by ljharb** [comment]
Everything observable is part of the public API.

**Comment by lencioni** [comment]
I think minor.

**Pull request merged**